Customization/T230/CodeSnippets/Activity2.1_Step2/RS301000.ts

```diff
@@ -0,0 +1,71 @@
+import {
+	PXScreen, createCollection, graphInfo,
+	viewInfo, createSingle,
+	PXView, PXFieldOptions, PXFieldState, controlConfig,
+	gridConfig, GridPreset, 
+	////////// The added code
+	PXActionState
+	////////// The end of added code
+} from "client-controls";
+
+@graphInfo({
+	graphType: "PhoneRepairShop.RSSVWorkOrderEntry",
+	primaryView: "WorkOrders"
+})
+export class RS301000 extends PXScreen {
+	@viewInfo({ containerName: "Work Order" })
+	WorkOrders = createSingle(RSSVWorkOrder);
+
+	@viewInfo({ containerName: "Repair Items" })
+	RepairItems = createCollection(RSSVWorkOrderItem);
+
+	@viewInfo({ containerName: "Labor" })
+	Labor = createCollection(RSSVWorkOrderLabor);
+}
+
+export class RSSVWorkOrder extends PXView {
+	OrderNbr: PXFieldState;
+
+	@controlConfig({allowEdit: true, })
+	CustomerID: PXFieldState<PXFieldOptions.CommitChanges>;
+	DateCreated: PXFieldState;
+	DateCompleted: PXFieldState;
+	Status: PXFieldState;
+
+	@controlConfig({rows: 2})
+	Description : PXFieldState<PXFieldOptions.Multiline>;
+
+	@controlConfig({allowEdit: true, })
+	ServiceID : PXFieldState<PXFieldOptions.CommitChanges>;
+
+	@controlConfig({allowEdit: true, })
+	DeviceID: PXFieldState<PXFieldOptions.CommitChanges>;
+	OrderTotal: PXFieldState;
+	Assignee: PXFieldState;
+	Priority: PXFieldState<PXFieldOptions.CommitChanges>;
+	InvoiceNbr: PXFieldState;
+}
+
+@gridConfig({
+	preset: GridPreset.Details
+})
+export class RSSVWorkOrderItem extends PXView {
+	RepairItemType: PXFieldState;
+	InventoryID: PXFieldState<PXFieldOptions.CommitChanges>;
+	InventoryID_description: PXFieldState;
+	BasePrice: PXFieldState;
+	////////// The added code
+	UpdateItemPrices: PXActionState;
```

Comment:
@MehranQadri , I don't see this in the list of Acumatica's code guidelines, but it seems that the usual practice is to list actions before the list of fields. Could you please change this for all screen classes and view classes?